src/triage.database/TriageDb.cs

```diff
@@ -18,7 +18,12 @@ public static class TriageDb
     {
         private static string s_connStr;
 
-        public static void Init(string connStr)
+        static TriageDb()
```

Comment:
Scott this is one change I made to TriageDb.cs. I call Init(...) explicitly here and paste in our connection string. I made Init(...) private so that no one is tempted to call it again.